src/applications/edu-benefits/1990s/content/OMBInfo.jsx

```diff
@@ -96,17 +96,17 @@ class OMBInfo extends React.Component {
           </button>
         </div>
 
-        <Modal
-          cssClass="va-modal-large"
-          contents={
-            this.props.children
-              ? this.props.children
-              : this.modalContents(resBurden)
-          }
+        <VaModal
```

Comment:
This modal is missing a title. For accessibility a title probably should be defined.